src/SeedsViewer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require_once 'src/Seed.php';
+require_once 'src/SeedsModel.php';
+
+class SeedsViewer
+{
+    // public static function displaySingleSeed(Seed $seed): string
+    // {
+    //         $output = '<div>';
+    //         $output .= "<h1>$seed->name</h1>";
+    //         $output .= "<p>$seed->family_name</p>";
+    //         $output .= "<p>$seed->species</p>";
+    //         $output .= "<p>$seed->description</p>";
+    //         $output .= "<img src='$seed->image' />";
+    //         $output .= '</div>';
+
+    //     return $output;
+    // }
+
+    public static function displayAllSeeds(array $seeds): string
+    {
+        $output = '';
+
+        foreach ($seeds as $seed) {
+            $output .= '<div>';
+            $output .= "<h1>$seed->name</h1>";
+            $output .= "<p>$seed->family_name</p>";
+            $output .= "<p>$seed->species</p>";
+            $output .= "<p>$seed->description</p>";
+            $output .= "<img src='$seed->image' />";
+            $output .= '</div>';
+        }
+
+        return $output;
+    }
```

Comment:
Here you should think about returning a basic message on the off chance that there are no seeds in the database